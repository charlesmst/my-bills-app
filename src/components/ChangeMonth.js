import React from "react";

export default function ChangeMonth({ currentDate }) {
  return <div>{currentDate.toISOString()}</div>;
}
