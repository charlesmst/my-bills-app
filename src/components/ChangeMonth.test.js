import ChangeMonth from './ChangeMonth'
it('should match snapshot',()=>{
    const wrapper = shallow(<ChangeMonth currentDate={new Date(2020,0,1,1,1,1)}/>);
    expect(wrapper).toMatchSnapshot();
})