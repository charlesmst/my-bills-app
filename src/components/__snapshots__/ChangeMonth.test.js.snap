// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  2020-01-01T03:01:01.000Z
</div>
`;
